src/routes/LandingPage.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+
+function LandingPage() {
+  return (
+    <>
```

Comment:
[NIT] can remove `<>` and `</>` and just use `<div>` and `</div>` as your opening and closing tags